Checkpoint 1

jamirhan opened the pull request.

jamirhan committed:
cmake

jamirhan committed:
empty class

jamirhan committed:
src change

Khabutdinov-Arslan commented on include/CoinsBuilder.h:
```diff
@@ -0,0 +1,10 @@
+
+
+class CoinsBuilder {
+
```
Нет контента

Khabutdinov-Arslan reviewed (changes requested):
В игре нет практически ничего кроме иерархии классов. К первому чекпойнту подразумевалось сделать больше.

EduardTkachenko committed:
Game creation update
update coin
update coin builder
update trap factory
update player
update traps
update server
update game
update game stats
update commutator
update environment
update graph
update messages
update tools

EduardTkachenko committed:
Game creation update
update coin
update coin builder
update trap factory
update player
update traps
update server
update game
update game stats
update commutator
update environment
update graph
update messages
update tools

EduardTkachenko committed:
update UML

EduardTkachenko committed:
Merge branch 'checkpoint_1' into dev

jamirhan committed:
Graph Creation Update

Khabutdinov-Arslan commented on src/Units/BestFriend.h:
```diff
@@ -0,0 +1,6 @@
+#pragma once
+#include "Unit.h"
+
+class BestFriend : public Unit {
```
Всё ещё пустовато, конечно

Khabutdinov-Arslan commented on src/Units/CCTV.h:
```diff
@@ -0,0 +1,6 @@
+#pragma once
+#include "Unit.h"
+
+class CCTV : public Unit {
```
Далее тоже

jamirhan committed:
Unit creation and Graph creation update

EduardTkachenko committed:
update coins creation and traps creation

EduardTkachenko committed:
merge remote-tracking branch 'origin/dev' into dev

EduardTkachenko committed:
update README

EduardTkachenko committed:
update UML

EduardTkachenko committed:
update README

jamirhan committed:
fixed errors from previous iteration

EduardTkachenko committed:
fir errors

Khabutdinov-Arslan reviewed (changes requested):
Добавьте библиотеки как сабмодули или просто не вносите в репозиторий хотя бы. Невозможно делать ревью 1500+ файлов

EduardTkachenko committed:
delete Eigen3 library files
